File: app/src/main/java/me/glide/testcontactupload/MainActivity.java
package me.glide.testcontactupload;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        findViewById(R.id.sync).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                ContactSyncService.startSync(MainActivity.this);
            }
        });
    }


}

File: app/src/main/java/me/glide/testcontactupload/MainActivity.kt
package me.glide.testcontactupload

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById(R.id.sync)!!.setOnClickListener { ContactSyncService.startSync(applicationContext) }
    }

}
